pywbem_mock/_mainprovider.py

```diff
@@ -786,14 +765,14 @@ def remove_namespace(self, namespace):
     #
     #   WBEM server request operation methods. These methods corresponde
     #   directly to the client methods in WBEMConnection.
-
+    #
     #   All the methods are named <methodname> and
     #   are responders that emulate the server response.
     #
     #   The API for these calls differs from the methods in _WBEMConnection
```

Comment:
Remove leading underscore in `_WBEMConnection`.

Reply:
Fixed. DONE